pandunia-loge.csv

```diff
@@ -137,7 +138,8 @@ Antigua|desh|||Antigua||Antigua||||||||||||||Antigua|Antigua
 Antigua e Barbuda|desh|AG||Antigua and Barbuda||Antigua y Barbuda||||||||||||||Antigua ja Barbuda|Antigua i Barbuda
 anus|||eng:fra:anus, por:ânus, spa:ano, deu:Anus, rus:анус (anus), tur:anüs|anus|anus|ano||анус||肛门|||||||||anuso|peräaukko (anus)|odbyt
 anusi||||anal||anal|||||||||||||anusa|anaalinen|analny, odbytni
-anzayisa||||remove (delete, dispose, erase)||quitar (eliminar, borrar)||||||||||||||poistaa (hävittää)|usuwać (usunąć, likwidować, zlikwidować, wymazywać, wymazać, pozbywać się, pozbyć się)
+anzayisa||||remove (dispose, dispell)||quitar||унестись (убирать)||||||||||||poistaa (hävittää)|usuwać (usunąć, likwidować, zlikwidować, wymazywać, wymazać, pozbywać się, pozbyć się)
```

Comment:
te no nida uza -is- sa yi loge. "anzaya" si basi.

Reply:
I was a little concerned that wouldn't work since **zayu** means "exist" and not "be created". but actually, given your recent clarifications to **-i** verbs, I think **zayu** _should_ mean "be created", and "exist" should be **zayi**.

Reply:
That's right. The system is more consistent now and **zayi** should (and does) mean "exist".